Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -170,40 +184,52 @@ open class MessageComposerInputAccessoryViewController<ExtraData: ExtraDataTypes
     // MARK: Button actions
     
     @objc func sendMessage() {
-        guard
-            let controller = controller,
-            let cid = controller.cid,
-            let text = composerView.messageInputView.textView.text,
+        guard let text = composerView.messageInputView.textView.text,
             !text.replacingOccurrences(of: " ", with: "").isEmpty
         else { return }
         
         switch state {
         case .initial:
             // TODO: Attachments
-            controller.createNewMessage(text: text)
-        case let .reply(messageToReply):
-            let messageController = controller.client.messageController(
-                cid: cid,
-                messageId: messageToReply.id
-            )
+            createNewMessage(text: text)
+        case .reply:
             // TODO:
             // 1. Attachments
             // 2. Should be inline reply after backend implementation.
-            messageController.createNewReply(text: textView.text, showReplyInChannel: true)
+            print("Inline reply sent.")
```

Comment:
Didn't you forgot to delete this `print`? :)

Reply:
Yeah, it's in `switch` so I can't live it empty.